db/migrate/20151012162248_change_user_search_index.rb

```diff
@@ -0,0 +1,29 @@
+class ChangeUserSearchIndex < ActiveRecord::Migration
+  def up
+#    remove_index :users, name: "users_idx_trgm_login_display_name"
+    add_column :users, :tsv, :tsvector
+
+    add_index :users, :tsv, using: "gin"
+
+    execute <<-SQL
+      CREATE TRIGGER tsvectorupdate BEFORE INSERT OR UPDATE
+      ON users FOR EACH ROW EXECUTE PROCEDURE
+      tsvector_update_trigger(
+        tsv, 'pg_catalog.english', login, display_name
+      );
+    SQL
+
+    now = Time.current.to_s(:db)
+    update("UPDATE users SET updated_at = '#{now}'")
+  end
+
+  def down
+  #  execute <<-SQL
```

Comment:
Thanks had to comment this out to fix a local broken migration. It's removed now.